### DISCUSSION TITLE
Font editor?

### DISCUSSION THREAD

**Discussion by Devnol** [Ralim/IronOS#1114] [Q&A] [answered]
I would like to contribute a translation to Greek, but there isn't a font for it. I have figured out how the font mapping works but is there any graphical way to create or edit a new font?

**Comment by Devnol** [comment, marked as answer]
Never mind, found it (/IronOS/Translations/BitmapEditor.html)

**Reply by Ralim** [reply]
Indeed that is probably the easiest to use, but if you have any issues don't be afraid to ask for help :) 

There are some others around as well who are skilled at creating graphics who can chime in if your unsure about anything

**Reply by Devnol** [reply]
Alright thanks! Now, regarding adding the font do I just make a new function in font_tables.py or what?

**Reply by Devnol** [reply]
Oh and add it to the elif chain I suppose

**Reply by Ralim** [reply]
Should just need to extend the font table dictionary to include the missing characters as the main change

**Reply by Devnol** [reply]
In that same file? Or is there anything else that needs to be updated?

**Reply by Ralim** [reply]
So if you make the JSON file for your language and open a PR the ci will run and then throw an error listing the missing characters.

The  you just need to add definitions for these to the font tables python file
